### DISCUSSION TITLE
How to add column in processes view?

### DISCUSSION THREAD

**Discussion by Makishima** [aristocratos/btop#563] [Q&A] [answered]
I want to add a CMD column, but I can't figure out how to do it. 
Based on the screenshots, it appears to be possible. 
Could someone please clarify the steps to add a column?

**Comment by deckstose** [comment]
What do you have you in mind? I don't think it is possible.

**Reply by Makishima** [reply]
I found out that I made a small mistake in the thread description. The Command column is actually present by default in the default view. However, I want to add it to the Tree view.

**Reply by deckstose** [reply, marked as answer]
No, that is not possible. But you can see the command in the tree view. It's inside brackets

**Comment by c33s** [comment]
so no way to add/see priorities and nice levels?

**Comment by W1T3C** [comment]
I have simlar question ... 
Can I monitor zpool mount in btop? if yes - how to do that ? When i add "/mnt/storage1" and "zstoragepool" into "Disk filter" - all disks dissaprear from monitoring window ...

root@backup6:~# df -h
Filesystem              Size  Used Avail Use% Mounted on
[...]
zstoragepool             88T   79T  8.3T  91% /mnt/storage1
[...]
